src/views/activity/daily/ActivityDaily.vue

```diff
@@ -21,7 +21,7 @@ div
             |  Next
           icon(name="arrow-right")
     div.p-1
-      b-select(:value="periodLength", :options="['day', 'week', 'month']",
+      b-select(:value="periodLength", :options="['day', 'week', 'month', 'year']",
```

Comment:
Oooof, going to get some HTTP timeouts until aw-server-rust becomes the default.

Reply:
Oops, commited by mistake.